src/hw/installOsuFirmware.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import type Transport from "@ledgerhq/hw-transport";
+import type { Observable } from "rxjs";
+import type { OsuFirmware } from "../types/manager";
+import ManagerAPI from "../api/Manager";
+
+export default (
+  transport: Transport<*>,
+  targetId: string | number,
+  firmware: OsuFirmware
+): Observable<*> => {
+  const params = {
+    targetId,
+    ...firmware,
```

Comment:
this is a bit nasty as we also send all the applications array as URL argument, which can be quite long... I guess it's safer/cleaner to just explode the keys we need (targetId, firmware, firmwareKey and perso)

Reply:
I'll take it

Reply:
seems like it's done already